go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/aerokube/selenoid-ui
 
+go 1.15
+
 require (
 	github.com/aandryashin/matchers v0.0.0-20161126170413-435295ea180e
-	github.com/aerokube/util v0.0.0-20181001032247-3a9b3f70da09
+	github.com/aerokube/util v0.0.0-20200816065139-4ca0222e7686
 	github.com/gorilla/websocket v1.4.0 // indirect
+	github.com/hashicorp/go-version v1.2.1 // indirect
 	github.com/koding/websocketproxy v0.0.0-20180716164433-0fa3f994f6e7
+	github.com/mitchellh/gox v1.0.1 // indirect
```

Comment:
I think it shouldn't be part of the go.mod, as its just a tool for ci

Reply:
Let's just revert updating util.

Reply:
Removed this stuff.